definitions/variable/afolu/land-cover-change.yaml

```diff
@@ -0,0 +1,65 @@
+- Forest Area Change:
+    description: Annual net change in total forest area between last period and this 
+      period (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Deforestation:
+    description: Annual gross deforestation between last period and this period 
+      (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Deforestation|Primary:
+    description: Annual gross loss of primary forest between last period and this period 
+      (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Deforestation|Secondary:
+    description: Annual gross loss of secondary forest between last period and this period 
+      (difference divided by period length)
+    unit: million ha/yr
+    tier: 3
+- Forest Area Change|Forest Expansion:
```

Comment:
Is this really something different than Afforestation? Given that this is in a variable referring explicitly to the area change...

Reply:
Yes, it is. Forest expansion also includes natural regeneration in secondary forest